int64 mission_id
RequestType request_type
---
mission mission_info